homie/Cargo.toml

```diff
@@ -5,10 +5,10 @@ authors = ["Andrew Walbran <qwandor@google.com>", "David Laban <alsuren@gmail.co
 edition = "2018"
 
 [dependencies]
-async-channel = "1.4.0"
-futures = "0.3"
-local_ipaddress = "0.1"
-log = "0.4"
-mac_address = "1.0"
+async-channel = "1.4.2"
+futures = "0.3.5"
+local_ipaddress = "0.1.3"
+log = "0.4.11"
+mac_address = "1.0.3"
```

Comment:
What is the advantage of specifying an exact version like this, rather than just a compatible version?

Reply:
This is what `cargo upgrade` does. 

I think the idea is that if someone already has a Cargo.lock with an older version of the dependency, and they install your lib, the version should automatically get bumped to one that's known to have all of the bug fixes etc. that your library has been tested against.